event/src/main.rs

```diff
@@ -1 +1,98 @@
-fn main() {}
+use std::sync::Arc;
+
+use clap::Parser;
+use database::Database;
+use poem::{
+    middleware::{Cors, Tracing},
+    EndpointExt, Result, Route,
+};
+use poem_openapi::{payload, LicenseObject, OpenApiService};
+
+use crate::config::Config;
+
+mod config;
+mod database;
+mod entities;
+mod error;
+mod routes;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    // This returns an error if the `.env` file doesn't exist, but that's not what we want
+    // since we're not going to use a `.env` file if we deploy this application.
+    dotenvy::dotenv().ok();
+
+    // Set up tracing
+    if std::env::var_os("RUST_LOG").is_none() {
+        #[cfg(debug_assertions)]
+        std::env::set_var("RUST_LOG", "poem=debug,identity=debug,auth0=info,sqlx=info");
+        #[cfg(not(debug_assertions))]
+        std::env::set_var("RUST_LOG", "poem=info,identity=info,auth0=info,sqlx=info");
```

Comment:
the `identity=debug` and `identity=info` here should be `event=debug` and `event=info` since the crate name is different now